src/flag_gems/runtime/backend/backend_utils.py

```diff
@@ -0,0 +1,19 @@
+import json
+import os
+from dataclasses import dataclass
+
+
+@dataclass
+class vendor_info_base:
```

Comment:
Please follow our code style and use CamaelCase for class name.

Reply:
Is this class designed to be a base class?
I don't see it bing subclassed?

Also, add documentation to clarify the intention of this class.